node-notes-cli/update.js

```diff
@@ -0,0 +1,14 @@
+const notes = require('./data.json');
+const fs = require('fs');
+
+const notesObj = notes.notes;
+
+function update(id, string) {
+  notesObj[id] = string;
+  const notesJSON = JSON.stringify(notes, null, 2);
+  fs.writeFile('data.json', notesJSON, err => {
+    if (err) throw err;
+  });
+}
```

Comment:
"User can update ..." functionality generally doesn't allow the user to create _new_ records by specifying an id that doesn't already exist. How would you prevent accidental note creation during an update?

Reply:
Added a logic check for the update function. confirms not undefined before writing to the id